tools/ci_build/github/linux/docker/Dockerfile.package_ubuntu_2004_gpu

```diff
@@ -46,7 +52,8 @@ RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/
     libnvonnxparsers-dev=${TRT_VERSION} \
     libnvonnxparsers10=${TRT_VERSION} \
     tensorrt-dev=${TRT_VERSION} \
-    libnvinfer-bin=${TRT_VERSION}
+    libnvinfer-bin=${TRT_VERSION} \
+    /usr/local/bin/install_cudnn.sh
```

Comment:
do we need to add another script?  would it just work to add the package name directly?
it's in the same repo as where it's installing the tensorrt libraries above it. 
https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cudnn9-cuda-12_9.1.1.17-1_amd64.deb